src/pages/guidelines/2x-grid/usage.mdx

```diff
@@ -0,0 +1,682 @@
+  <AnchorLink>Fit for purpose</AnchorLink>
+  <AnchorLink>Content hierarchy</AnchorLink>
+  <AnchorLink>Continuity and contrast</AnchorLink>
+  <AnchorLink>Grid influencers</AnchorLink>
+  <AnchorLink>Style models</AnchorLink>
+  <AnchorLink>Gutter modes</AnchorLink>
+  <AnchorLink>Mixing gutter modes</AnchorLink>
+  <AnchorLink>Using grids in Figma</AnchorLink>
+</AnchorLinks>
+
+## UI galleries
+
+For years Carbon users have been asking for a gallery of product UI screens for
+reference and inspiration. There are several ongoing initiatives to catalog
+screens and we’ll continue to add to this list as more come online.
+
+<Row className="resource-card-group">
+  <Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="UI screens (IBMers only)"
+    title="Security"
+    href="https://pages.github.ibm.com/security/security-design/department/tools-and-resources/visual-libraries/libraries/"
+    >
+
+<IbmSecurity size={32} />
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## Fit for purpose
+
+On every page there is a story that ends with a desired action–one that enables
+a user to achieve a goal or reach an outcome. Proper layouts pull everything
+together to help users form a mental model of the story, identify relevant
+content, and pursue their objective.
+
+Always start with the content and user goals. Consider the purpose of the
+content and the best type of experience to help the user reach their objective.
+For instance, is it more of a long-form reading experience or is the user
+scanning a dashboard or completing a multi-step task? Each type of experience
+and purpose will require a different layout.
+
+## Content hierarchy
+
+Hierarchy helps users (viewer, reader, audience) to navigate complex concepts
+without getting confused or lost, and ultimately find what they are looking for.
+Thoughtful application of type styles, components, and patterns will allow
+content to be communicated and prioritized in different ways. Pay attention also
+to the size and proximity of content pieces within a component and between
+components.
+
+### Basic scaffolding
+
+In the Overview section, we talked a bit about common screen regions—like the UI
+Shell, the content area and dialogs. Now we also need to address several common
+layout patterns within the content area in IBM products.
+
+These repeatable structures lower the cognitive burden for users, by reinforcing
+key alignments throughout the UI. Below are some examples of structural layout
+modules that are used widely across products. For more detailed documentation
+about structural layout modules, check out
+[IBM Product Layouts](https://pages.github.ibm.com/cdai-design/pal/layouts/library/)
+on the Carbon for IBM Products PAL.
+
+![Grid scaffolding illustration](images/usage/01_basic_scaffolding.png)
+
+<Caption>
+  Although gutter width depends on the product, here are some common examples of
+  how product designers use a consistent leadspace size with the 2x Grid to
```

Comment:
Is "leadspace" a unique IBM term? I'm not able to find a solid definition for it. I personally don't know what it refers to. Maybe the page heading? If so, would "page heading" be a more universal way to describe this?